ArduCopter/control_throw.cpp

```diff
@@ -95,9 +95,14 @@ void Copter::throw_run()
         pos_control.set_speed_z(BRAKE_MODE_SPEED_Z, BRAKE_MODE_SPEED_Z);
         pos_control.set_accel_z(BRAKE_MODE_DECEL_RATE);
 
-        // initialise the demanded height to 3m above the throw height
-        // we want to rapidly clear surrounding obstacles
-        pos_control.set_alt_target(inertial_nav.get_altitude() + 300);
+        // initialise the demanded height to 3m above the throw height or 1 meter below the drop height
```

Comment:
If the intention is for this to be used for air drop, then 1m does not seem like enough margin given baro and other height estimation errors.